scripts/build_docs.py

```diff
@@ -0,0 +1,34 @@
+import logging
```

Comment:
I'm not sure we actually need this anymore. ReadTheDocs does the necessary.

In particular `sphinx-apidoc` is not necessary, since I checkin the skeleton `.rst` files.

Reply:
you mean the entire file?

Reply:
Yes. ReadTheDocs just cares about the docs directory, and that it has a `conf.py`

Reply:
Wonderful...

Reply:
deleted